libs/common/src/sql-db-connection/migrations/1734616272101-AddFeeRecordCorrectionTransientFormDataTable.ts

```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFeeRecordCorrectionTransientFormDataTable1734616272101 implements MigrationInterface {
+  name = 'AddFeeRecordCorrectionTransientFormDataTable1734616272101';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE TABLE "FeeRecordCorrectionTransientFormData" (
+                "userId" nvarchar(255) NOT NULL,
+                "correctionId" int NOT NULL,
+                "formDataSerialized" nvarchar(1000) NOT NULL,
+                "lastUpdatedAt" datetime2 NOT NULL CONSTRAINT "DF_6bf1a666c5c54f26a80a93cb477" DEFAULT getdate(),
```

Comment:
@ChristianMcc these not null constraints were generated with the same name used on the request transient form data table, I assume since they are generated using table name and column name at the time of generation. I simply replaced the random looking part with the starts of new guids, this works locally and I think is fine, but since we are determining this ourselves we could also go for more descriptive names. Just wanted to flag in case you think there could be any issues with this?

Reply:
Hey, yeah a while ago we did think about naming these as they are just generated guids to be unique in name. I think naming them is a good idea and no issue with it - would recommend keeping the same identifier at the start though (e.g. a default keeping DF or unique constraint keeping UQ etc.)